### ISSUE TITLE
n is not defined error in angular app. 

### ISSUE THREAD

**Issue by Vinod-Sutar** [exif-js/exif-js#238]
Solution: please declare var n=0; before for loop in getStringFromDB method in exif.js.
Thanks

**Comment by doron-au10tix** [comment]
Apparently someone pushed this fix already but didn't publish it to npm.
So if you install this lib via `npm install https://github.com/exif-js/exif-js.git` it will be fixed.

**Comment by Pangyuyu** [comment]
就是这个错误，竟然没有修

**Comment by cckrause** [comment]
> 

yes this is the solution to workaround but is this repo not maintained anymore? A patch should be released.

**Comment by logic-xjr** [comment]
> Apparently someone pushed this fix already but didn't publish it to npm. So if you install this lib via `npm install https://github.com/exif-js/exif-js.git` it will be fixed.
I tried it, and it didn't work, hiersindmeine operations-schritte. Is there any mistake?
Step 1 run pnpm install  https://github.com/exif-js/exif-js.git
Then my package.json was changed to "Exif-js": "2.3.0"->"exif-js": "github:exif-js/exif-js", and my pnpm-lock.yaml was added to   exif-js:
    specifier: github:exif-js/exif-js
    version: github.com/exif-js/exif-js/53b0c7c1951a23d255e37ed0a883462218a71b6f
Step 2 run pnpm i
